catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/OperationsMetacardSupport.java

```diff
@@ -140,6 +140,7 @@ void generateMetacardAndContentItems(List<ContentItem> incomingContentItems,
                 metacardMap.put(metacard.getId(), metacard);
 
                 ContentItem generatedContentItem = new ContentItemImpl(metacard.getId(),
+                        StringUtils.isNotEmpty(contentItem.getQualifier()) ? contentItem.getQualifier() : "",
```

Comment:
❓ `null` is a valid value for `qualifier` for the `ContentItemImpl` constructor. Any reason we've added that check here?